lib/sendgrid-ruby.rb

```diff
@@ -1,5 +1,6 @@
 require_relative 'sendgrid/client'
 require_relative 'sendgrid/version'
+require_relative 'sendgrid/helpers/ip_management/ip_management'
```

Comment:
@clee Added ip_management here, re: https://github.com/sendgrid/sendgrid-ruby/pull/236#pullrequestreview-95961002

Reply:
Awesome! I'm seeing Travis checking it now.

@brokenthumbs Have you signed the CLA?

Reply:
What's the approach here for dealing with an old commit, that does not have an email address associated with the commit?

Reply:
Yup, that seems to be the issue. The easiest solution would be for you to use `git rebase --interactive`, mark the commit that has the wrong email for 'edit' instead of 'pick', and then use 'git commit --amend --reset-author' on that commit. (If you need more help to understand all of that, no problem, I'm on Freenode and OFTC as clee. Hit me up there!)